source/transactions/tests/unified/retryable-commit-handshake.yml

```diff
@@ -0,0 +1,111 @@
+description: "retryable commitTransaction on handshake errors"
+
+schemaVersion: "1.4"
+
+runOnRequirements:
+  - minServerVersion: "4.2"
+    topologies: [replicaset, sharded, load-balanced]
+    serverless: "forbid"
+    auth: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      observeEvents: [commandStartedEvent, connectionCheckOutStartedEvent]
+      uriOptions: { retryReads: true, retryWrites: true, maxPoolSize: 5 }
```

Comment:
Since the related spec change says `commitTransaction` is retryable "regardless of whether retryWrites is set on the MongoClient or not", would it make sense to explicitly specify `retryWrites: false` here?

Beyond that, my earlier comment about default values and `maxPoolSize` applies here.

Reply:
I agree with the retryWrites false, done!